pusherable/templatetags/pusherable_tags.py

```diff
@@ -3,29 +3,30 @@
 
 register = template.Library()
 
+
 @register.simple_tag
 def pusherable_script():
-    return "<script src=\"//js.pusher.com/2.2/pusher.min.js\" type=\"text/javascript\"></script>"
+    return ('<script src="//js.pusher.com/3.0/pusher.min.js" '
+            'type="text/javascript"></script>')
 
 
 @register.simple_tag
-def pusherable_subscribe(event, instance):
-
-    channel = u"{model}_{pk}".format(
-        model=instance._meta.model_name,
-        pk=instance.pk
-    )
+def pusherable_subscribe(instance):
+    """
+    Channel: <model_name>_<model_primary_key>
+    """
+    channel = '{model_name}_{model_pk}'.format(
+        model_name=instance._meta.model_name, model_pk=instance.pk)
 
     return """
```

Comment:
How likely is it that the user might need to update this template? Seems like this should be a rendered template file instead of a string in the view.